action.yml

```diff
@@ -0,0 +1,55 @@
+inputs:
+  targets:
+    description: 'depviz target (format: \"org1/repo1 org1/repo2 org2 repo1\")' 
+    required: true
+  branch: 
+    description: 'the branch where to push the generated files' 
+    default: 'depviz-generate'
+  gen-json: 
+    description: 'generate json output'
+    default: 'false'
+  gen-png: 
+    description: 'generate png output'
+    default: 'false'
+  gen-csv:
+    description: 'generate csv output'
+    default: 'false'
+
+
+runs:
+  using: 'composite'
+  steps:
+  - uses: actions/checkout@v3
+
+  - run: mkdir db
+    shell: bash
+
+  - uses: actions/setup-go@v4
+
+  - run: mkdir depviz-outputs
```

Comment:
The mkdir should be `mkdir -p $(dirname ${{ inputs.png-output}})  $(dirname ${{ inputs.csv-output}}). $(dirname ${{ inputs.json-output}}) `